addons/event/i18n/nl.po

```diff
@@ -1588,8 +1588,8 @@ msgstr ""
 "                        Hallo <t t-out=\"object.name or 'Guest'\"/>,<br/><br/>\n"
 "                        We zijn blij om je registratie voor het evenement \n"
 "                        <t t-if=\"is_online\">\n"
-"                            <a t-att-href=\"object.event_id.website_url\" style=\"color:#875A7B;text-decoration:none;font-weight:bold;\" t-out=\"object.event_id.name or ''\">OpenWood Collectie Online Onthulling</a>\n"
-"                        </t> te bevestigen\n"
+"                            <a t-att-href=\"object.event_id.website_url\" style=\"color:#875A7B;text-decoration:none;font-weight:bold;\" t-out=\"object.event_id.name or ''\">OpenWood Collectie Online Onthulling</a> te bevestigen\n"
```

Comment:
Instead of moving the "te bevestigen" inside the if, shouldn't we just move it after the else, so we do have it present for both? (We could also modify the template structure to avoid any confusion on translators but maybe not in stable 🤔 )

Like:

```suggestion
"                            <a t-att-href=\"object.event_id.website_url\" style=\"color:#875A7B;text-decoration:none;font-weight:bold;\" t-out=\"object.event_id.name or ''\">OpenWood Collectie Online Onthulling</a>\n"
```